test/UnitTests/MSTest.Core.Unit.Tests/Assertions/AssertTests.IsInstanceOfTypeTests.cs

```diff
@@ -20,53 +20,90 @@ namespace Microsoft.VisualStudio.TestPlatform.TestFramework.UnitTests;
 public partial class AssertTests
 {
     [TestMethod]
-    public void InstanceOfTypeShouldFailWhenValueIsNull()
+    public void IsInstanceOfType_WhenValueIsNull_Fails()
     {
         static void action() => TestFrameworkV2.Assert.IsInstanceOfType(null, typeof(AssertTests));
         ActionUtility.ActionShouldThrowExceptionOfType(action, typeof(TestFrameworkV2.AssertFailedException));
     }
 
     [TestMethod]
-    public void InstanceOfTypeShouldFailWhenTypeIsNull()
+    public void IsInstanceOfType_WhenTypeIsNull_Fails()
     {
         static void action() => TestFrameworkV2.Assert.IsInstanceOfType(5, null);
         ActionUtility.ActionShouldThrowExceptionOfType(action, typeof(TestFrameworkV2.AssertFailedException));
     }
 
     [TestMethod]
-    public void InstanceOfTypeShouldPassOnSameInstance()
+    public void IsInstanceOfType_OnSameInstance_Passes()
     {
         TestFrameworkV2.Assert.IsInstanceOfType(5, typeof(int));
     }
 
     [TestMethod]
-    public void InstanceOfTypeShouldPassOnHigherInstance()
+    public void IsInstanceOfType_OnHigherInstance_Passes()
     {
         TestFrameworkV2.Assert.IsInstanceOfType(5, typeof(object));
     }
 
     [TestMethod]
-    public void InstanceNotOfTypeShouldFailWhenValueIsNull()
+    public void IsNotInstanceOfType_WhenValueIsNull_Fails()
     {
         Action action = () => TestFrameworkV2.Assert.IsNotInstanceOfType(null, typeof(AssertTests));
```

Comment:
This test is broken. You're not invoking the `Action`.

Reply:
Test was present before (not introduced in this PR).

Reply:
Ping @engyebrahim